database/src/main/resources/db/migration/V1.42.0__NOTIF-491_initial_manual_vacuum.sql

```diff
@@ -0,0 +1,6 @@
+-- This script should allow Postgres to run index-only scans on the following tables immediately, without waiting for
```

Comment:
We need two migration scripts because Flyway does not allow transactional (`ALTER TABLE`) and non-transactional (`VACUUM`) statements in the same file.